src/stores/connectors/keys/list.go

```diff
@@ -0,0 +1,33 @@
+package keys
+
+import "context"
+
+func (c Connector) List(ctx context.Context) ([]string, error) {
+	ids := []string{}
+	keysRetrieved, err := c.db.Keys().GetAll(ctx)
```

Comment:
Alternatively we could read IDs directly from DB for optimization proposes

Reply:
is this query excluding soft_deleted items ?

Reply:
won't have time but I created a ticket: https://app.zenhub.com/workspaces/orchestrate-5ea70772b186e10067f57842/issues/consensys/quorum-key-manager/241